docs/requirements.txt

```diff
@@ -1,4 +1,5 @@
 absl-py
+ipython==8.6.0
```

Comment:
Also add a TODO to fix ipython3 issue and remove this pinning

Reply:
Done.